CoralBase/cmt/requirements

```diff
@@ -32,7 +32,7 @@ macro_append use_cppflags '' target-vc9 ' /D_CRT_SECURE_NO_WARNINGS'
 macro_append use_cppflags '' target-vc9 ' /D_SCL_SECURE_NO_WARNINGS'
 
 # Test -std=c++0x for gcc 4.5.1 as in CMS (see task #18131)
-# Disable "auto_ptr is deprecated" warning for gcc 4.5.
+# Disable "unique_ptr is deprecated" warning for gcc 4.5.
```

Comment:
this change was not required. The comment here is related to gcc 4.5 so I think the old comment makes better sense

Reply:
Oh, yes thanks for catching this! I'll update.